install_config/install/disconnected_install.adoc

```diff
@@ -0,0 +1,550 @@
+
+You now need to link:../../install_config/install/docker_registry.adoc[create the internal Docker registry].
+
+[[disconnected-post-installation-changes]]
+== Post-Installation Changes
+
+In one of the previous steps, the S2I images were imported into the Docker
+daemon running on one of the OpenShift Master hosts. In a connected
+installation these images would be pulled from Red Hat’s registry on demand.
+Since the Internet is not available to do this, the images must be made
+available in another Docker registry.
+
+OpenShift provides an internal registry for storing the images that are built
+as a result of the S2I process, but it can also be used to hold the S2I builder
+images. The following steps assume you did not customize the service IP subnet
+(172.30.0.0/16) or the Docker registry port (5000).
+
+[[disconnected-re-tag-s2i-builder-images]]
+=== Re-tag S2I Builder Images
+
+. On the Master host where you imported the S2I builder images, obtain the
+service address of your docker registry that you installed on the master:
++
+----
+# export REGISTRY=$(oc get service docker-registry -t '{{.spec.clusterIP}}{{"\n"}}')
+----
+
+. Next, tag all of the builder images before pushing them into the OpenShift Docker registry:
++
+----
+# docker tag registry.access.redhat.com/jboss-amq-6/amq-openshift $REGISTRY:5000/openshift/amq-openshift
+# docker tag registry.access.redhat.com/jboss-eap-6/eap-openshift $REGISTRY:5000/openshift/eap-openshift
+# docker tag registry.access.redhat.com/jboss-webserver-3/tomcat7-openshift $REGISTRY:5000/openshift/tomcat7-openshift
+# docker tag registry.access.redhat.com/jboss-webserver-3/tomcat8-openshift $REGISTRY:5000/openshift/tomcat8-openshift
+# docker tag registry.access.redhat.com/rhscl/mongodb-26-rhel7 $REGISTRY:5000/openshift/mongodb-26-rhel7
+# docker tag registry.access.redhat.com/rhscl/mysql-56-rhel7 $REGISTRY:5000/openshift/mysql-56-rhel7
+# docker tag registry.access.redhat.com/rhscl/perl-520-rhel7 $REGISTRY:5000/openshift/perl-520-rhel7
+# docker tag registry.access.redhat.com/rhscl/php-56-rhel7 $REGISTRY:5000/openshift/php-56-rhel7
+# docker tag registry.access.redhat.com/rhscl/postgresql-94-rhel7 $REGISTRY:5000/openshift/postgresql-94-rhel7
+# docker tag registry.access.redhat.com/rhscl/python-27-rhel7 $REGISTRY:5000/openshift/python-27-rhel7
+# docker tag registry.access.redhat.com/rhscl/python-34-rhel7 $REGISTRY:5000/openshift/python-34-rhel7
+# docker tag registry.access.redhat.com/rhscl/ruby-22-rhel7 $REGISTRY:5000/openshift/ruby-22-rhel7
+# docker tag registry.access.redhat.com/openshift3/nodejs-010-rhel7 $REGISTRY:5000/openshift/nodejs-010-rhel7 
+----
+
+[[disconnected-create-an-admin-user]]
+=== Create an Administrative User
+
+Pushing the Docker images into OpenShift’s Docker registry requires a user with administrator privileges. Because the default OpenShift system administrator does not have a standard authorization token, they cannot be used to log in to the Docker registry.
```

Comment:
@tnguyen-rh Using 'they' should be ok here. It's the 'he/she' vs 'they' debate... They is accepted as being a gender neutral thing: http://stylepedia.net/#sect-Red_Hat_Technical_Publications-Writing_Style_Guide-Grammar-Gender_References